SRC/dlassq.f90

```diff
@@ -39,13 +39,24 @@
 !>    ( scl**2 )*smsq = x( 1 )**2 +...+ x( n )**2 + ( scale**2 )*sumsq,
 !>
 !> where  x( i ) = X( 1 + ( i - 1 )*INCX ). The value of  sumsq  is
-!> assumed to be non-negative and  scl  returns the value
-!>
-!>    scl = max( scale, abs( x( i ) ) ).
+!> assumed to be non-negative.
 !>
 !> scale and sumsq must be supplied in SCALE and SUMSQ and
 !> scl and smsq are overwritten on SCALE and SUMSQ respectively.
 !>
+!> If scale * sqrt( sumsq ) > tbig then
+!>    we require:   scale >= sqrt( TINY*EPS ) / sbig   on entry,
+!> and if scale * sqrt( sumsq ) < tsml then
+!>    we require:   scale <= sqrt( HUGE ) / ssml       on entry,
```

Comment:
Maybe it's nitpicking but documented like this isn't `scale=1`, `sumsq=0` excluded?